libobs/obs-output.c

```diff
@@ -166,11 +166,11 @@ obs_output_t *obs_output_create(const char *id, const char *name,
 	return NULL;
 }
 
-void obs_output_update_video(obs_output_t *output)
+void obs_output_update_av(obs_output_t *output)
 {
-	if (output && output->info.update_video) {
-		output->info.update_video((void *)output->context.data,
-					  obs_get_video());
+	if (output && output->info.update_av) {
+		output->info.update_av((void *)output->context.data,
+					  obs_get_video(), obs_get_audio());
 	}
 }
```

Comment:
This function should check for both update_av and update_video, preferring update_av if it is found.

Reply:
update_video was replaced by update_av